code.py

```diff
@@ -1,32 +1,46 @@
+from botocore.exceptions import ClientError
+from pathlib import Path, PureWindowsPath, PurePosixPath
 from sqlalchemy import create_engine
 from time import time
+from typing import Dict
 import pandas as pd
-import os, toml
+import boto3, glob, os, toml
 
-def read_config(config_file):
-    """Read TOML file
+CONFIG_FILE_NAME = 'config.toml'
+OUTPUT_DIRECTORY = 'parquet'
+FILE_EXT = '.parquet'
+AWS_S3_BUCKET = 'ensembl-genome-data-parquet'
 
+def read_config(config_file_name: str) -> Dict[str, str]:
+    '''
     Read TOML file containing the DB info
 
     Args:
-        config_file (str): TOML configuration file name
+        config_file: TOML configuration file name
 
     Returns:
-        Dictionary containing the parsed data.
+        Dictionary containing the parsed data
 
     Raises:
-        FileNotFoundError: If `config_file` doesn't exist.
-    """
+        FileNotFoundError: If `config_file` doesn't exist
+    '''
 
     try:
```

Comment:
If we can't read the file shouldn't we just fail and kill the process right away? Also you said in the docs this raises a file not found error but we're trapping it here if I am reading this correctly

Reply:
@andrewyatz Updated the PR with a new commit which should handle this, please check and re-review